packages/shortcuts-extension/src/components/ShortcutItem.tsx

```diff
@@ -358,9 +386,14 @@ export class ShortcutItem extends React.Component<
     return this.props.shortcut.keys[key].map(
       (keyBinding: string, index: number) => (
         <div className="jp-Shortcuts-ShortcutKeysContainer" key={index}>
-          <div className="jp-Shortcuts-ShortcutKeys">
+          <button
```

Comment:
Why is this a button? This element corresponds to individual key in a sequence, it has no button role.